docs/self-managed/components/orchestration-cluster/core-settings/configuration/logging.md

```diff
@@ -0,0 +1,67 @@
+---
+id: logging
+title: Logging
+description: Configuration settings for logging in the Camunda Platform 8 orchestration cluster components.
+---
+
+This page covers logging configuration for the Camunda Platform 8 orchestration cluster components. Logging is handled via the Log4j2 framework and can be customized to suit your needs, including output formats, logging levels, and runtime log level adjustments.
+
+## Log4j2 configuration
+
+Each orchestration cluster component includes a default Log4j2 configuration file that can be modified to adjust logging behavior:
```

Comment:
There's only one Orchestration Cluster application now, so it has only one logging configuration file, not one per application.